Superpowered/OpenSource/SuperpoweredIOSAudioIO.mm

```diff
@@ -149,11 +149,14 @@ - (void)dealloc {
 - (void)onMediaServerReset:(NSNotification *)notification { // The mediaserver daemon can die. Yes, it happens sometimes.
     if (![NSThread isMainThread]) [self performSelectorOnMainThread:@selector(onMediaServerReset:) withObject:nil waitUntilDone:NO];
     else {
+        BOOL restart = audioUnitRunning;
         if (audioUnit) AudioOutputUnitStop(audioUnit);
         audioUnitRunning = false;
         [[AVAudioSession sharedInstance] setActive:NO error:nil];
         [self resetAudio];
-        [self start];
+        if (restart) {
```

Comment:
This is the second part of this fix: https://github.com/ResonantCavity/voloco_ios/pull/3521